ERCS/erc-7662.md

```diff
@@ -0,0 +1,103 @@
+---
+eip: 7662
+title: AI Agent NFTs
+description: A specification for NFTs that represent AI Agents.
+author: Greg Marlin (@0marleymarl)
```

Comment:
```suggestion
author: Greg Marlin (@marleymarl)
```
This should be your GitHub username

Reply:
Good catch @abcoathup  LOL

Reply:
Was going to say, nice catch, thanks - fixed.